meeting-notes/2018/2018-12-03--js-core-dev-team-weekly.md

```diff
@@ -0,0 +1,201 @@
+# ⚡️ⒿⓈ Core Dev Team Weekly Sync 🙌🏽 December 3, 2018
+
+- **Lead:** @alanshaw
+- **Notetaker:** @jacobheun
+- **Attendees:**
+  - @vmx
+  - @hugomrdias
+  - @vasco-santos
+  - @Elexy
+  - @mcolina	
+  - @litzenberger
+  - @achingbrain
+- **Recording:** https://youtu.be/zvJljf0sYf8
+
+## Agenda
+
+- Ask everyone to put their name into the list of attendees
+- Round of updates
+  - What have you accomplished since the last Weekly?
+  - Were there any blockers? If so, which ones? Is it still blocked? Why?
+  - What is the next important thing you should focus on?
+- Ask for general questions. Could be things like:
+  - I'm stuck with something, I don't know who to ask. Who knows who to ask?
+  - Who can help me with xyz?
+- Plan this week
+  - Select issues to work on
+- Review remaining issues if there is time left
+
+## Notes
+
+### Week Update
+
+@vasco-santos
+
+- Done:
+  - IPNS over Pubsub
+    - Fixed reviews across all open PRs [js-ipfs#1559](https://github.com/ipfs/js-ipfs/pull/1559)
+  - IPNS over DHT
+    - Ready to review [ipfs/js-ipfs#1725](https://github.com/ipfs/js-ipfs/pull/1725)
+    - Interop tests [ipfs/interop/pull/47](https://github.com/ipfs/interop/pull/47)
+  - IPFS Interop files fix
+    - [ipfs/interop#48](https://github.com/ipfs/interop/pull/48)
+  - DHT Stress tests
+    - Intensive lookup and churn configurable [iptb-dht-test](https://github.com/vasco-santos/iptb-dht-test)
+- Blocked: N/A
+- Next:
+  - `IPNS over Pubsub` and `IPNS over DHT` merged
+  - libp2p daemon client for testbed and interop
+  - get [libp2p/js-libp2p-kad-dht#39](https://github.com/libp2p/js-libp2p-kad-dht/pull/39) mergeable
+  - update IPNS spec PR
+
+@vmx
+ - Done:
+   - [js-cid](https://github.com/multiformats/js-cid) is now part of the multiformats org (instead of IPLD)
+   - [multibase](https://github.com/multiformats/multibase) wording improved. Thanks @stebalien for rewording things properly
+   - Started to implement new JS IPLD API specs (https://github.com/ipld/js-ipld/pull/185, https://github.com/ipld/interface-ipld-format/pull/50)
+ - Blocked: Nope
+ - Next:
+  - Keep implementing new specs to uncover flaws
+  - Keep improving multiformat specs as I encounter issues
+
+@alanshaw
+- Done:
+     - Added CID version agnostic-ness interop tests
+         - <img width="377" alt="screenshot 2018-11-27 at 15 49 23" src="https://user-images.githubusercontent.com/152863/49093555-43034180-f25c-11e8-885c-4633ab2c55ed.png">
+         - https://github.com/ipfs/interop/pull/46
```

Comment:
@eingenito @kevina here are the CID Base32 Interop tests I mentioned in the go core dev call :)